# >>>
# >>>
# >>> import psutil
# >>>
# >>>
# >>> psutil.cpu_count()
# 22
# >>> psutil.cpu_count(logical=False)
# 16
# >>> psutil.virtual_memory()
# svmem(total=33629761536, available=16050499584, percent=52.3, used=17579261952, free=16050499584)
# >>> psutil.disk_usage('/')
# sdiskusage(total=1022229811200, used=99104739328, free=923125071872, percent=9.7)
# >>>
# >>> psutil.sensors_temperatures()
# Traceback (most recent call last):
#   File "<python-input-10>", line 1, in <module>
#     psutil.sensors_temperatures()
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
# AttributeError: module 'psutil' has no attribute 'sensors_temperatures'
# >>> psutil.sensors_battery()
# sbattery(percent=100, secsleft=<BatteryTime.POWER_TIME_UNLIMITED: -2>, power_plugged=True)
# >>> psutil.pids()
# [0, 4, 308, 352, 544, 972, 984, 1108, 1132, 1448, 1552, 1628, 1712, 1724, 1844, 1884, 2024, 2112, 2120, 2132, 2216, 2236, 2264, 2280, 2288, 2372, 2420, 2452, 2484, 2520, 2552, 2624, 2644, 2708, 2724, 2732, 2760, 2784, 2928, 3080, 3108, 3212, 3296, 3372, 3380, 3396, 3404, 3556, 3592, 3656, 3708, 3716, 3752, 3800, 3880, 3928, 3944, 4104, 4140, 4192, 4248, 4256, 4304, 4340, 4416, 4432, 4440, 4536, 4544, 4616, 4748, 4776, 4884, 4976, 5056, 5064, 5224, 5340, 5452, 5484, 5496, 5516, 5532, 5600, 5688, 5744, 5764, 5772, 5792, 5796, 5808, 5840, 5864, 5876, 5884, 5892, 5900, 5908, 5920, 5928, 5936, 5948, 5956, 5968, 5976, 5980, 6004, 6016, 6036, 6044, 6056, 6068, 6280, 6420, 6480, 6524, 6568, 6724, 6936, 7096, 7116, 7132, 7368, 7404, 7464, 7480, 7688, 8304, 8396, 8476, 8508, 8524, 8740, 8756, 8764, 8876, 8880, 8896, 8912, 9044, 9108, 9208, 9256, 9408, 9592, 9864, 9916, 9992, 10156, 10172, 10176, 10188, 10496, 10544, 10588, 10636, 10676, 10820, 10832, 11136, 11192, 11364, 11396, 11440, 11536, 11644, 11916, 11944, 12064, 12116, 12264, 12276, 12356, 12380, 12732, 12768, 12972, 13048, 13212, 13224, 13232, 13236, 13516, 13680, 13704, 13788, 14012, 14372, 14620, 15248, 15324, 15368, 15432, 15436, 15820, 15956, 16132, 16236, 16316, 16396, 16512, 16564, 16580, 16640, 16708, 16836, 17056, 17116, 17172, 17568, 18036, 18052, 18060, 18368, 18376, 18400, 18580, 18600, 18916, 19008, 19056, 19096, 19188, 19320, 19384, 19484, 19732, 19776, 19900, 20008, 20128, 20168, 20180, 20192, 20200, 20208, 20336, 20764, 21132, 21140, 21292, 21764, 21972, 22028, 22104, 22156, 22216, 22368, 22480, 22788, 23088, 23364, 23644, 24312, 24456, 24488, 24544, 24860, 24936, 25104, 25108, 25140, 25152, 25380, 25392, 25504, 25620, 25852, 25932, 26592, 27180, 27432, 27652, 27712, 27784, 27796, 27804, 27972, 28104, 28156, 28212, 28492, 28528, 29204, 29368, 29456, 29536, 29632, 29640, 29680, 29860, 29892, 29920, 30120, 30272, 30288, 30364, 30456, 30644, 30676, 30744, 30912, 31040, 31052, 31224, 31284, 31500, 31520]
# >>>  p = psutil.Process(31224)
#   File "<python-input-13>", line 1
#     p = psutil.Process(31224)
# IndentationError: unexpected indent
# >>> p = psutil.Process(31224)
# >>>
# >>> p.status()
# 'running'
# >>> p.memory_info()
# pmem(rss=28307456, vms=14397440, num_page_faults=225932, peak_wset=77893632, wset=28307456, peak_paged_pool=647104, paged_pool=607808, peak_nonpaged_pool=187808, nonpaged_pool=42712, pagefile=14397440, peak_pagefile=16855040, private=14397440)
# >>> p.cpu_times()
# pcputimes(user=4.953125, system=9.5, children_user=0.0, children_system=0.0)
# >>> import os
# >>> os.getpid()
# 8876
